datasets/indonlu/indonlu.py

```diff
@@ -0,0 +1,631 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The IndoNLU benchmark is a collection of resources for training, evaluating, and analyzing natural language understanding systems for Bahasa Indonesia"""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import textwrap
+
+import six
+
+import datasets
+
+
+_INDONLU_CITATION = """\
+@inproceedings{wilie2020indonlu,
+title = {{IndoNLU}: Benchmark and Resources for Evaluating Indonesian Natural Language Understanding},
+authors={Bryan Wilie and Karissa Vincentio and Genta Indra Winata and Samuel Cahyawijaya and X. Li and Zhi Yuan Lim and S. Soleman and R. Mahendra and Pascale Fung and Syafri Bahar and A. Purwarianti},
+booktitle={Proceedings of the 1st Conference of the Asia-Pacific Chapter of the Association for Computational Linguistics and the 10th International Joint Conference on Natural Language Processing},
+year={2020}
+}
+"""
+
+_INDONLU_DESCRIPTION = """\
+The IndoNLU benchmark is a collection of resources for training, evaluating, \
+and analyzing natural language understanding systems for Bahasa Indonesia.
+"""
+
+_INDONLU_HOMEPAGE = "https://www.indobenchmark.com/"
+
+_INDONLU_LICENSE = "https://raw.githubusercontent.com/indobenchmark/indonlu/master/LICENSE"
+
+
+class IndoNluConfig(datasets.BuilderConfig):
+    """BuilderConfig for IndoNLU"""
+
+    def __init__(
+        self,
+        text_features,
+        label_column,
+        label_classes,
+        train_url,
+        valid_url,
+        test_url,
+        citation,
+        **kwargs,
+    ):
+        """BuilderConfig for IndoNLU.
+
+        Args:
+          text_features: `dict[string, string]`, map from the name of the feature
+            dict for each text field to the name of the column in the txt/csv/tsv file
+          label_column: `string`, name of the column in the txt/csv/tsv file corresponding
+            to the label
+          label_classes: `list[string]`, the list of classes if the label is categorical
+          train_url: `string`, url to train file from
+          valid_url: `string`, url to valid file from
+          test_url: `string`, url to test file from
+          citation: `string`, citation for the data set
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(IndoNluConfig, self).__init__(version=datasets.Version("1.0.0", ""), **kwargs)
+        self.text_features = text_features
+        self.label_column = label_column
+        self.label_classes = label_classes
+        self.train_url = train_url
+        self.valid_url = valid_url
+        self.test_url = test_url
+        self.citation = citation
+
+
+class IndoNlu(datasets.GeneratorBasedBuilder):
```

Comment:
```suggestion
class Indonlu(datasets.GeneratorBasedBuilder):
```
This must be the camel case version of the dataset script name `indonlu`

Reply:
Thanks for the review @lhoestq!
I've made some changes from your review. Hopefully, this is the last! 🤗